source/_integrations/imap.markdown

```diff
@@ -91,6 +91,14 @@ headers:
 
 The `event_type` for the custom event should be set to `imap_content`. The configuration below shows how you can use the event data in a template `sensor`.
 
+If the default maximum message size (2048) to be used in events is too small for your needs, then this maximum size setting can be increased(advanced mode is required).
+
+<div class='note warning'>
+
+Increasing the default maximum message size (2048) could have a negative impact on performance as event data is also logged by the `recorder`. If the total event data size is exceeds the maximum event size (32168) the event will be skipped.
```

Comment:
```suggestion
Increasing the default maximum message size (2048 bytes) could have a negative impact on performance as event data is also logged by the `recorder`. If the total event data size exceeds the maximum event size (32168 bytes), the event will be skipped.
```
I added bytes everywhere to be more explicit about the unit. But maybe there is a reason you did not add this?